### ISSUE TITLE
Error opening data.yaml file

### ISSUE THREAD

**Issue by vp1993** [PRBonn/bonnet#34]
I'm trying to use the pretrained model to generate masks for a video. I followed the instructions on how to run it 
`python ./cnn_video.py small.mp4 -p cnn_video_pb.py`

and it gives me this error: 

`INTERFACE:
Video to infer:  
Log dir:  /tmp/net_predict_log
model path cnn_video_pb.py
model type iou
data yaml:  None
net yaml:  None
train yaml:  None
Verbose?:  False
----------

Opening default data file data.yaml from log folder
Error opening data yaml file...
`

**Comment by tano297** [comment]
Hi there, you need to use the path to the trained model to infer, not ``cnn_video_pb.py``. The links are in the main [readme](https://github.com/PRBonn/bonnet/blob/master/README.md#pre-trained-models)
